src/bg_cache_worker.ts

```diff
@@ -0,0 +1,151 @@
+// Hack to be able to find all our modules with relative paths
+process.env.NODE_PATH = __dirname;
+// tslint:disable-next-line:no-var-requires
+require('module').Module._initPaths();
+
+import * as Arc from "@daostack/arc.js";
+import * as aws from 'aws-sdk';
+import { normalize } from "normalizr";
+import promisify = require("es6-promisify");
+import HDWalletProvider from "./lib/truffle-hdwallet-provider";
+import Web3 = require("web3");
+
+import * as arcActions from "./actions/arcActions";
+import { initialState as arcInitialState, IArcState, IDaoState, TransactionStates, IVoteState } from "./reducers/arcReducer";
+import * as schemas from "./schemas";
+import Util from "./lib/util";
+
+const network = Arc.Config.get('network');
+
+if (process.env.NODE_ENV == 'production') {
+  // Use Infura on production
+  const infuraNetwork = network == 'live' ? 'mainnet' : network;
+
+  // The default account in reading from Infura, I'm not totally sure why this is needed (but it is), or what account it should be
+  const mnemonic = process.env.DEFAULT_ACCOUNT_MNEMONIC;
+
+  const provider = new HDWalletProvider(mnemonic, "https://" + infuraNetwork + ".infura.io/UeW8cwaou03qFgsAHoDP");
```

Comment:
Where is `DEFAULT_ACCOUNT_MNEMONIC` defined?

Reply:
Heroku

Reply:
So, this is the secret key for the node that's running in infura?

Reply:
Seems weird that we can just access it

Reply:
Oh, it's actually the secret key for the account that interacts with the infura rpc.

Reply:
no its a mnemonic for a default account. the key is actually embedded in the URL, but really that should also be an ENV variable :) ill do that. what do you mean weird that we can access it? thats how Heroku handles environment variables. how else would it work?

Reply:
A mnemonic is just a "pretty printer" secret key.
nevermind, all is good :)